metrics/expvar/expvar.go

```diff
@@ -75,9 +81,35 @@ type histogram struct {
 	hist *hdrhistogram.WindowedHistogram
 
 	name   string
+	count  metrics.Counter
 	gauges map[int]metrics.Gauge
 }
 
+// Name returns the name of the histogram
+func (h *histogram) Name() string {
+	return h.name
+}
```

Comment:
Name isn't part of the [Histogram definition](https://godoc.org/github.com/go-kit/kit/metrics#Histogram); what consumes that? Should it be added to the definition?